ktor-server/ktor-server-core/src/jsAndWasmSharedMain/kotlin/io/ktor/server/request/ApplicationReceiveFunctions.jsAndWasmShared.kt

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2014-2025 JetBrains s.r.o and contributors. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package io.ktor.server.request
+
+@PublishedApi
+internal actual val DEFAULT_FORM_FIELD_LIMIT: Long
```

Comment:
Nit. We've created `nonJvm` source-set recently. Maybe this implementation could be moved there.